src/Pages/mihyeon/Login/Login.js

```diff
@@ -1,13 +1,83 @@
-import React, { Component } from 'react';
+import React from "react";
+import { Link, withRouter } from "react-router-dom";
+import "./Login.scss"
+
+class Login extends React.Component {
+
+    state = {
+        id: '',
+        password: '',
+        validate: false,
+        iscolor: false,
+    }
+
+    handleChange = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
```

Comment:
```suggestion
    handleChange = (e) => {
    const { name, value } = e.target;
        this.setState({
            [name]: value
```
계산된 속성명 사용 잘 해주셨네요 미현님! 
이런식으로 함수 내부에서 비구조화 할당 가능합니다!